### PR TITLE
feat: [SRM-11300]: Modification in AppDynamics for metic threshold

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary

Modification in AppDynamics for metic threshold
On updating  Metric pack in AppDynamics health source , values in custom metric form and metric threshold should persist.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vasuki-kalluri — DISMISSED** [review]

**Review by praneshg239 — APPROVED** [review]

**Comment by deepesh-ui** [comment]
retrigger standards

**Review by praneshg239 — APPROVED** [review]

File: cypress/support/85-cv/monitoredService/constants.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const environmentistCall =
   '/cv/api/monitored-service/environments?routingId=accountId&accountId=accountId&orgIdentifier=default&projectIdentifier=project1'
 
 export const monitoredServiceListCall =
-  '/cv/api/monitored-service?routingId=accountId&offset=0&pageSize=10&accountId=accountId&orgIdentifier=default&projectIdentifier=project1&servicesAtRiskFilter=false'
+  '/cv/api/monitored-service?routingId=accountId&offset=0&pageSize=10&accountId=accountId&orgIdentifier=default&projectIdentifier=project1&filter=&servicesAtRiskFilter=false'
```
Comment (amitsh14) [inline cypress/support/85-cv/monitoredService/constants.ts:74]: No value is assigned to the filter param. Is this allowed?

**Review by amitsh14 — APPROVED** [review]

File: cypress/support/85-cv/monitoredService/constants.ts (hunk 1 above)
Comment (deepesh-ui) [reply cypress/support/85-cv/monitoredService/constants.ts:74]: yes , this is to unblock the current cypress failure , @vasuki-kalluri  will be working on the fix

**Comment by deepesh-ui** [comment]
retrigger coverage

**Pull request merged**